web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -513,7 +513,7 @@ import {get_string_diff} from "../../src/util";
                 e.preventDefault();
                 this.select(e);
             }
-            this.suppressKeyPressRepeat = !~$.inArray(pseudo_keycode, [40, 38, 9, 13, 27]);
+            this.suppressKeyPressRepeat = ![40, 38, 9, 13, 27].includes(pseudo_keycode);
```

Comment:
I'm not entirely sure what madman would combine `!` and `~`, but it's not immediately obvious this is equivalent given the return type of `inArray`...

This commit could probably benefit from a commit message explaining the equivalency logic.

Reply:
`indexOf` and jQuery `inArray` return `-1` for a missing element. `0` is the only integer that’s falsy, so `-1` is the only integer whose bitwise complement is falsy. This is one of those conventions that was obvious to JS devs back in the pre-ES2016 era before `.includes` existed.

Reply:
Added that to the commit message